components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -102,8 +103,12 @@ object Nimbus : NimbusApi {
         context: Context,
         onExperimentUpdated: ((activeExperiments: List<EnrolledExperiment>) -> Unit)? = null
     ) {
-        // Set the name of the native library
-        System.setProperty("uniffi.component.nimbus.libraryOverride", "megazord")
+        // Set the name of the native library so that we use
+        // the appservices megazord for compiled code.
+        System.setProperty(
+            "uniffi.component.nimbus.libraryOverride",
+            System.getProperty("mozilla.appservices.megazord.library", "megazord")
+        )
```

Comment:
This was the simplest thing I could come up with for working with custom megazords.

Reply:
Works for me